docs/source/release_notes.rst

```diff
@@ -363,12 +483,11 @@ unique, some options to work around the error are below.
 - If the error is resulting from a batch job, add **ignore_errors:true** to the task and capture the output into a variable and extract the job ID with
   a regular expression and then use ``zos_job_output`` to display the DD without the non-printable character such as the DD **JESMSGLG**.
 
-Availability
-------------
+-- ``zos_copy`` and ``zos_job_submit`` - supports Jinja2 templating which is essential for handling tasks that require advanced file modifications such as JCL.
```

Comment:
the above line may be another merge error/issue

Reply:
Fixed the release notes